CondFormats/HcalObjects/src/HBHEDarkening.cc

```diff
@@ -105,7 +105,7 @@ float HBHEDarkening::degradation(float intlumi, int ieta, int lay) const {
   //accumulate degradation over years
   float response = 1.0;
   std::string yearForLumi = getYearForLumi(intlumi);
-  assert(yearForLumi.size());
+  assert(!yearForLumi.empty());

```

Comment:
This one doesn’t seem to be the same.

Reply:
why not ?
isn't `assert(s.size())` the same as `assert(s.size() != 0)` which is the same as `assert(! s.empty())` ?

Reply:
Sorry, I didn’t explain myself well. This change is unlike all the other changes in the pull request.

Reply:
Ah yes: I think this comes from a pre-existing check, that was somehow missed so far.
It appears in some of the other PRs as well.